src/main/java/com/krglow/booking/app/util/JsonUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.krglow.booking.app.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class JsonUtil {
```

Comment:
Ta klasa jest używana tylko w testach? Jeśli tak to czy nie powinna być w pakietach testowych? A jeśli ma być użwana w kodzie aplikacji to czy nie powinien to być bean?